homeassistant/util/json.py

```diff
@@ -84,7 +84,10 @@ def save_json(
             dump = _orjson_default_encoder
             json_data = _orjson_default_encoder(data)
     except TypeError as error:
-        msg = f"Failed to serialize to JSON: {filename}. Bad data at {format_unserializable_data(find_paths_unserializable_data(data, dump=dump))}"
+        msg = (
+            f"Failed to serialize to JSON: {filename}. Bad data at"
+            f" {format_unserializable_data(find_paths_unserializable_data(data, dump=dump))}"
+        )
         _LOGGER.error(msg)
```

Comment:
Side note: Maybe we should only raise and not also log?

Reply:
Good point, made a note to get back to that later.